habushu-maven-plugin/src/main/java/org/technologybrewery/habushu/exec/PyenvCommandHelper.java

```diff
@@ -25,29 +27,31 @@ public class PyenvCommandHelper {
     private File workingDirectory;
 
     public PyenvCommandHelper(File workingDirectory) {
-	this.workingDirectory = workingDirectory;
+        this.workingDirectory = workingDirectory;
     }
 
     /**
      * Returns a boolean value indicating whether pyenv is installed.
      */
     public boolean isPyenvInstalled() {
-	try {
-	    execute(Arrays.asList("--version"));
-	} catch (Throwable e) {
-	    return false;
-	}
-	return true;
+        try {
+            String foundVersion = executeWithDebugLogging(Arrays.asList("--version"));
+            logger.debug("Found " + foundVersion);
```

Comment:
I: just made some of the debugging with pyenv debug level so it's a more crisp in the info output what is happening that people typically care about